app/collins/models/User.scala

```diff
@@ -20,7 +20,29 @@ abstract class User(val username: String, val password: String) {
     case false => None
   }
   def isEmpty(): Boolean = username.isEmpty && password.isEmpty && roles.isEmpty
-  def canSeePasswords(): Boolean = Permissions.please(this, Permissions.Feature.CanSeePasswords)
+  def canSeeEncryptedTags(): Boolean = {
+    // want to rename canSeePasswords to canSeeEncryptedTags
```

Comment:
Picky, but this comment is a bit hard to decipher at first glance. Maybe something like
```
// We want to rename canSeePasswords to canSeeEncryptedTags but want to keep
// backward compatibility. So we use canSeeEncryptedTags if it is set, else fall
// back to canSeePasswords. 
// 
// Permissions default to true, so we also need to test if the permission is
// configured, and only use it if it is.
```